app/src/main/java/org/thunderdog/challegram/ui/SettingsController.java

```diff
@@ -630,12 +632,12 @@ public void setValuedSetting (ListItem item, SettingView view, boolean isUpdate)
     items.add(new ListItem(ListItem.TYPE_SEPARATOR));
     items.add(new ListItem(ListItem.TYPE_SETTING, R.id.btn_privacySettings, R.drawable.baseline_lock_24, R.string.PrivacySettings));
     items.add(new ListItem(ListItem.TYPE_SEPARATOR));
+    items.add(new ListItem(ListItem.TYPE_SETTING, R.id.btn_stickerSettingsAndEmoji, R.drawable.deproko_baseline_stickers_filled_24, R.string.StickersAndEmoji));
+    items.add(new ListItem(ListItem.TYPE_SEPARATOR));
     items.add(new ListItem(ListItem.TYPE_SETTING, R.id.btn_themeSettings, R.drawable.baseline_palette_24, R.string.ThemeSettings));
     items.add(new ListItem(ListItem.TYPE_SEPARATOR));
     items.add(new ListItem(ListItem.TYPE_SETTING, R.id.btn_tweakSettings, R.drawable.baseline_extension_24, R.string.TweakSettings));
     items.add(new ListItem(ListItem.TYPE_SEPARATOR));
-    items.add(new ListItem(ListItem.TYPE_SETTING, R.id.btn_stickerSettings, R.drawable.deproko_baseline_stickers_filled_24, R.string.Stickers));
-    items.add(new ListItem(ListItem.TYPE_SEPARATOR));
     items.add(new ListItem(ListItem.TYPE_SETTING, R.id.btn_languageSettings, R.drawable.baseline_language_24, R.string.Language));
```

Comment:
Fuck u.. i hop karma always with u all and for u my love take back this pain
Jika ini balasan utk semua pgorbananku aku stop dan kau blh bersama betina yg kau nak dan kesakitan,kesulitan,kesusahan jgk kehancuran akan bersama kamu slamanya.. Aamiin☝️ ya ALLAH aku d zholimi olh org yg aku percaya aku x redha dan ikhlas dgn kebohongan yg berdrama dlm hidupku smoga mereka semua rasa apa yg aku rasa.. kun fa ya kun  Aamiin.